src/Adapter/Product/Image/Repository/ProductImageRepository.php

```diff
@@ -120,7 +130,8 @@ public function getImages(ProductId $productId): array
             ->from($this->dbPrefix . 'image', 'i')
```

Comment:
I'm afraid that this will strike back when QA will test multi-store behavior. I don't see anything related to multi-store and I'm almost 100% sure that `ObjectModel` won't handle everything automatically if you're using queries like this 😢

Reply:
We didn't handle multi-store for now, because we lacked time to handle it all for 1.7.8 and the beta version of the page That's nearly no Product command handle multi shop for now, the AddProductCommand is an exception, we'll see if it creates bug but as long as we have only one shop it should be fine

All the multishop management (in product page) will be added for 1.7.9

Reply:
Ok, understood. I hope we'll have legacy product page, as long as new one will not be fully working with multi-store, and - of course - this should be properly communicated, and maybe new product page shouldn't be available if merchant has multi-store enabled 🤔

Reply:
The new page will be hidden in 1.7.8 shops, and merchants will need to enable it from inside the BO to see it 😉  so they'll know it's an experimental feature, not a production-ready page.

That being said, is there, in the current Product page, a management of images for multistore 😮  ? I see how you can handle multistore configuration in database, but images multistore management seems like a scary subject.

Reply:
It's more or less automatic, when you edit an image it updates the association with the context shops (this much is handled in the upload command for now)

To answer your question @kpodemski
- yes the legacy page will still be present and be used as the default page
- the new page will be disabled by default but will be "activatable" via our new "Feature flag" feature
- even with the new page activated the legacy one is still usable (by default) it just enables you access to the new ones via new buttons
- for multistore I think you're right, to avoid any unexpected behaviour we should block the product page in multistore context (or as soon as multi stores exist?)